olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -51,6 +51,10 @@ metadata:
     repository: https://github.com/openshift-knative/serverless-operator
     support: Red Hat
     olm.skipRange: '>=1.13.0 <1.14.0'
+  labels:
+    operatorframework.io/arch.amd64: supported
```

Comment:
How these labels are used?

Reply:
They determine on which arches an operator is displayed on the web console operatorhub.